CHANGELOG.md

```diff
@@ -7,6 +7,20 @@ Unreleased
 ### SDK Bugs
 
 
+Release v1.8.0 (2022-09-13)
+================================
+### SDK Enhancements
+* Replace error type assertions with errors.As in DefaultFormattingStrategy [#PR 353](https://github.com/aws/aws-xray-sdk-go/pull/353)
+* Dummy segments don't need cancel go routine [#PR 365](https://github.com/aws/aws-xray-sdk-go/pull/365)
+* Strip X-Amz-Security-Token from SQL URIs [#PR 367](https://github.com/aws/aws-xray-sdk-go/pull/367)
+* Added OTel Go links to readme [#PR 369](https://github.com/aws/aws-xray-sdk-go/pull/369)
+* Added segment creation in instrumentation example [#PR 370](https://github.com/aws/aws-xray-sdk-go/pull/370)
```

Comment:
Changes that don't impact the behavior of the SDK should not be included in the changelog